api/nodes/views.py

```diff
@@ -1282,6 +1282,8 @@ class NodeFilesList(generics.ListAPIView, WaterButlerMixin, ListFilterMixin, Nod
         base_permissions.TokenHasScope,
     )
 
+    ordering = ('materialized_path',)  # default ordering
```

Comment:
I'll leave it to you to decide best course of action, but adding `date_modified` to each class seems a fine approach.